.github/CODEOWNERS

```diff
@@ -1 +1,18 @@
-* @iHiD
+# JS files
+*.js      @iHiD @kntsoriano @erikschierboom
+
+# Ruby files
```

Comment:
Is the `Rakefile` usual to require review?

Reply:
Good shout. Thanks.